content/md/what-is/what-about-reference-entities.md

```diff
@@ -0,0 +1,57 @@
+---
+id: what-about-reference-entities
+themes: first-steps, manage-your-reference-entities
+title: What about **reference entities**?
+popular: true
+ee-only: true
+related: manage-reference-entities, enrich-your-reference-entity-records
+---
+
+# What's the use case?
+
+Let's take an example of a product page.  
+![Reference entity use case](../img/what-about-reference-entities.png)
+
+There are many types of product information such as text, images...  
+Some information are shared between different products (such as the definition of care instructions, colors or brands).  
+These shared information can be complex with their own attributes (eg a label, a logo, a description or photos for the brand).  
```

Comment:
"This shared information" instead of "these" I guess, not sure... Maybe we should ask Camille B. to validate it.  ;)

Reply:
"e.g." instead of "eg" I guess... idem

Reply:
Yes it's "this" thanks + "e.g." fixed